package.json

```diff
@@ -10,7 +10,8 @@
   },
   "scripts": {
     "start": "parallelshell 'bundle exec jekyll serve --watch --baseurl ''' 'gulp watch'",
-    "build": "gulp build"
+    "build": "gulp build",
+    "cf-link": "npm link cf-pagination"
```

Comment:
As in capital-framework, this script will only work for cf-pagination. You can add arguments to `npm run <script>` commands, right?